packages/mobile/src/escrow/actions.ts

```diff
@@ -1,15 +1,15 @@
 import { PhoneNumberHashDetails } from '@celo/identity/lib/odis/phone-number-identifier'
 import BigNumber from 'bignumber.js'
 import { FeeInfo } from 'src/fees/saga'
-import { SHORT_CURRENCIES } from 'src/geth/consts'
 import { TransactionContext } from 'src/transactions/types'
+import { Currency } from 'src/utils/currencies'
 
 export interface EscrowedPayment {
   senderAddress: string
   recipientPhone: string
   recipientIdentifier: string
   paymentID: string
-  currency: SHORT_CURRENCIES
+  currency: Currency
```

Comment:
I don't think it's necessary because this gets overwritten on app open, and also the escrow reducer doesn't handle the rehydrate action

Reply:
When there's no explicit `REHYDRATE` action handled in the reducer, the state is still rehydrated. Unless it's in the blacklist: https://github.com/celo-org/wallet/blob/a1e6dda13ba3321bb659a0427454c88d518d6164/packages/mobile/src/redux/store.ts#L25
The `REHYDRATE` action is mostly used to tweak / adapt the rehydrated state.

Though this specific state is quickly updated on app open, I think we should migrate it to be on the safe side. We'd need to double all usage, but without looking it's possible this would lead to an undefined value somewhere.

Reply:
Added migration